packages/babel-compat-data/data/native-modules.json

```diff
@@ -2,6 +2,7 @@
   "es6.module": {
     "chrome": "61",
     "and_chr": "61",
+    "deno": "1.0",
```

Comment:
I believe this is what causes CI to fail. We can introduce `deno` support in a semver-minor PR.

Reply:
Do we need an issue for it?

Reply:
The deno support version comes from `@mdn/browser-compat-data`, however, the `compat-table` does not support deno yet, so currently we are blocked by https://github.com/kangax/compat-table/issues/1651.